scripts/hello-world.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo 'Hello world script'
+echo '{"msg":"Hello '$1'!","error":{"message":"nil"}}'
```

Comment:
this will probably serve as a goto example of how to output json from a .sh step
should we use

cat <<EOF
{"msg": "Hello $1!"}
EOF

? Less escaping tricks.

Also the error object here is a bit confusing, it resembles some form of contract for error handling but I see no other reference to it in the plugin code or doc.

Reply:
As a best practice,you should never try to create structured data by string concatenation, it doesn't work well and can conduct to injection issues.

You should use tool who knows how to handle JSON like jq
```
$ echo '"John" Doe' | jq -R '{"msg":("Hello " + . + "!"), "error":{"message":null}}'
{
  "msg": "Hello \"John\" Doe!",
  "error": {
    "message": null
  }
}
```

Reply:
I agree /w both of you but I moved this script in a dedicated testing folder to highligth the fact that is not an example, just a dumb script wrote *for testing purpose*. So did really we need changes on it? WDYT?

@Alkorin I also agree about using `jq` for this kind of stuff but I don't want to embed a system dependency in each testing env (CI/laptop/desktop...), it works fine in this way *for testing purpose*.